### PR TITLE
Feature/gitlab support

### PR DESCRIPTION
**Pull request by jlcooper**
A patch to support GitLab hosted repositories as an alternative to GitHub.

### CODE REVIEW

**Comment by jlcooper** [comment]
The failure of the Travis-CI setup seems to be an issue with perlbrew versions 5.20 and below, rather than an issue with the actual code.

File: lib/Minilla/Project.pm [hunk 1]
```diff
@@ -524,16 +524,16 @@ sub extract_git_info {
     if ( my $registered_url = `git config --get remote.origin.url` ) {
         $registered_url =~ s/\n//g;
         # XXX Make it public clone URL, but this only works with github
-        if ($registered_url !~ m{^file://} && $registered_url =~ /github\.com/) {
-            my ($user, $repo) = $registered_url =~ m{
-                github\.com
+        if ($registered_url !~ m{^file://} && $registered_url =~ /(?:github|gitlab)\.com/) {
+            my ($gitService, $user, $repo) = $registered_url =~ m{
```
Comment (skaji) [inline lib/Minilla/Project.pm]: gitService -> git_service

Comment (jlcooper) [reply lib/Minilla/Project.pm]: I've switched the variable name to the rest of the codebase style as suggested.

**Comment by skaji** [comment]
@jlcooper Thanks. Released https://metacpan.org/release/SKAJI/Minilla-v3.1.6

**Pull request merged**